models/shipping-method/src/shipping-method/generator.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ClientLogging,
+  LocalizedString,
+  Reference,
+} from '@commercetools-test-data/commons';
+import { sequence, fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TShippingMethod } from './types';
+
+//https://docs.commercetools.com/api/projects/taxCategories#taxcategory
```

Comment:
Big whoops on my part
```suggestion
//https://docs.commercetools.com/api/projects/shippingMethods#shippingmethod
```